star_schema_db.sql

```diff
@@ -0,0 +1,70 @@
+DROP DATABASE star;
+CREATE DATABASE star;
+\c star;
+
+CREATE TABLE fact_table (
+employee_number INTEGER REFERENCES employee_logs(employee_number),
+office_code INTEGER REFERENCES office_logs(office_code),
+customer_number INTEGER REFERENCES customer_logs(customer_number),
+order_number INTEGER REFERENCES order_logs(order_number),
+sales_rep_employee_number INTEGER REFERENCES employee_logs(employee_number),
+product_code VARCHAR REFERENCES product_logs(product_code),
+quantity_in_stock INTEGER,
+buy_price FLOAT,
+_m_s_r_p FLOAT,
+quantity_ordered INTEGER NOT NULL,
+price_each FLOAT NOT NULL,
+order_line_number INTEGER,
+credit_limit INTEGER,
```

Comment:
This is really a fact of the customers, not the orders! Remember, the fact table should correspond to one type of "business process" or "event".